CHANGELOG.unreleased.md

```diff
@@ -8,6 +8,7 @@
 > Users must be able to say: “Nice enhancement, I'm eager to test it”
 
 - [VM export] Feat export to `ova` format (PR [#6006](https://github.com/vatesfr/xen-orchestra/pull/6006))
+- [Backup] Merge multiple VHDs at once (PR [#6184](https://github.com/vatesfr/xen-orchestra/pull/6184))
```

Comment:
As usual, I wonder if this makes sense for users as this change is not visible.

I you think it's a good idea, at least highlight how that will benefit them, otherwise remove it.